index.js

```diff
@@ -1,7 +1,73 @@
 // Iteration 1: Names and Input
 
+let hacker1 = "Lucas";
+console.log("driver's name: " + hacker1);
+let hacker2 = "Juan";
+console.log("navigator's name: " + hacker2);
 
 // Iteration 2: Conditionals
+  function which() {
```

Comment:
```suggestion
function which() {
```
Es por pijo eee jejeje que me gusta la indentacion mucho! :D